modules/tinymce/tools/docs/tinymce.EditorManager.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * This is the base class for the link:../tinymce.root/[`tinymce`] global, a factory for link:../tinymce.editor/[`tinymce.Editor`] instances.
+ *
+ * It is not generally used directly - the global link:../tinymce.root/[`tinymce`] is the only one expected to be created.
```

Comment:
This doesn't really make sense, what's trying to be created? I'm not sure what it's trying to say though to be able to make a suggestion sorry.

Reply:
I'm trying to say we don't expect customers to create an instance of this class. We create one, which is the `tinymce` global, and that's all.

Reply:
I've changed to "The global `tinymce` is the only one expected to be created." Is that better?